### ISSUE TITLE
"Mute Microphone" control does not work when bound to Mouse 4

### ISSUE THREAD

**Issue by sdarkwell** [henkelmax/simple-voice-chat#259]
**Bug description**
When the "Mute Microphone" control is bound to Mouse 4, it does not work. It appears to bind correctly, but neither pressing and releasing or pressing and holding the Mouse 4 button has any effect. Mouse 4 _does_, however, work as the "Push To Talk" button.

Note: Mouse 4, by default, is the Microsoft Windows "Back" button, but does normally does not have that functionality within Minecraft. Not certain whether or not this may be related to the issue. 

**Steps to reproduce the issue**
1. Start a game/join a world
2. Go to Game Menu
3. Click Options...
4. Click Controls...
5. Under the "Voice Chat" header, click "Mute Microphone"
6. Press your Mouse 4 button to bind the button to the mouse
7. Click Done
8. Click Done
9. Press your Mouse 4 button
10. Note that the microphone is not muted

**Expected behavior**
Microphone should mute/unmute when the Mouse 4 button is pressed

**Log files**
[Client and Server Log Files](https://gist.github.com/sdarkwell/4a01b7fb1764fedb997be52854d7d792/)

**Versions**
- Minecraft version: 1.16.5
- Forge/Fabric version: Forge v36.1.32
- Mod version: voicechat-forge-1.16.5-1.6.15

**Other mods**
None

**Screenshots (Optional)**
N/A

**Comment by henkelmax** [comment]
I tested it and it works perfectly fine. Please make sure you are actually using voice activation. PTT has no mute button.

**Comment by sdarkwell** [comment]
**Hello @henkelmax,**

Apologies for re-opening the issue, but I have confirmed that I am in fact using Voice Activation, not Push to Talk. Keyboard keys work without issue, but mouse buttons do not.

Having tested it further, it seems that pressing the mouse button _does_ toggle the mute state, but it is not refreshed until a keyboard key is pressed. So, if you press the mute button on the mouse and press any keyboard keys, you will remain in the previous mute state until a keyboard button is pressed, at which time the mute state will refresh. Pressing other mouse buttons does not, however, refresh the mute state.

For example:
1. Begin unmuted
2. Speak (Others can hear you)
3. Press mute mouse button
4. Speak (Other can still hear you)
5. Press any other button on the keyboard
6. Speak (You are now muted)

Both my girlfriend and I were able to recreate this behavior using difference mice.

**Thank you and be well!**
- S.
